karabo_data/geometry2/__init__.py

```diff
@@ -359,10 +368,23 @@ def plot_data_fast(self, data, axis_units='px', frontview=True):
         frontview : bool
           If True (the default), x increases to the left, as if you were looking
           along the beam. False gives a 'looking into the beam' view.
+        ax : `~matplotlib.axes.Axes`, list of Axes, optional
+          Parent axes that will be used to draw the image.
```

Comment:
I don't think this can be a list of axes here, because we call `ax.imshow()`.

Reply:
Also, it's not the *parent* axes in this context, just the axes. It's the parent for the colorbar axes.

Reply:
Yes, of course! That was me being lazy while copying that from the matplotlib docstring.